src/hypothesis/extra/pytestplugin.py

```diff
@@ -19,6 +19,9 @@
 import pytest
 
 PYTEST_VERSION = tuple(map(int, pytest.__version__.split('.')[:3]))
+LOAD_PROFILE_OPTION = '--hypothesis-profile'
+
+PYTEST_VERSION = tuple(map(int, pytest.__version__.split('.')))
```

Comment:
That addition seems wrong, as it's already defined above (with the `[:3]` introduced recently)